### ISSUE TITLE
Plugin sometimes stops working

### ISSUE THREAD

**Issue by jjwhawaii** [nicoduj/homebridge-gogogate2#51]
Hello, sometimes I get this message, and the plugin stops working:

[37m[11/19/2023, 2:13:30 AM] [39m[36m[Jeff Wong][39m WARNING - refreshDoor - GarageDoorOpenerGarage - operation was in progress and has timedout or no status retrieval

I turned the plugin into a child bridge, which I think helps, but still it occasionally freezes:

[37m[11/22/2023, 4:43:17 PM] [39m[36m[Gogogate2][39m ERROR - refreshDoor - Refreshing status failed - undefined
[37m[11/22/2023, 4:43:17 PM] [39m[36m[Gogogate2][39m ERROR - handleError - timeout connecting to gogogate, check gogogate connectivity

Here is my config:

            "name": "Gogogate2",
            "gogogateIP": "192.168.4.32",
            "username": 
            "password": 
            "refreshTimer": 600,
            "maxWaitTimeForOperation": 35,
            "refreshTimerDuringOperation": 10,
            "_bridge": {
                "username": "0E:38:8C:AC:5E:21",
                "port": 59834
            },
            "platform": "GogoGate2"

Would it help to change the refresh timer?

Thanks for all your work on this Plugin!
-Jeff

**Comment by nicoduj** [comment]
Hi, seems a bit strange, it is like your gogogate becomes unreachable. Does it have a static IP ? When it happens, do you need to restart the bridge?

**Comment by jjwhawaii** [comment]
Yes static IP.  I'm not sure why it wasn't working back in November, but
the plugin has been working great for the past 2 months!  Thanks for your
help.

--
Jeff
